src/web/middleware/sentry.rs

```diff
@@ -147,6 +149,11 @@ where
                 Some(e) => {
                     let apie: Option<&ApiError> = e.as_error();
                     if let Some(apie) = apie {
+                        if apie.is_conflict() {
+                            if let Some(state) = sresp.request().app_data::<Data<ServerState>>() {
+                                Metrics::from(state.as_ref()).incr("db.conflict");
```

Comment:
Oh one nit: could we call this `"storage.conflict"`, everything else uses the storage prefix

Reply:
yeah, just copied what was there, but no reason not to get a jump on normalizing this crap.